frontend/views/containers/payments/PaymentsList.vue

```diff
@@ -54,30 +144,99 @@ export default ({
 .c-payments {
   margin-top: 1rem;
 
-  th,
-  td {
-    &:first-child {
-      width: 41%;
+  &.c-is-todo {
+    th.c-th-check-all {
+      width: 1.125rem;
+
+      .checkbox {
+        margin-right: 0.2rem;
+      }
+    }
+
+    th.c-th-who {
+      width: 65%;
+
+      @include tablet {
+        width: 27%;
+      }
+    }
+
+    th.c-th-method {
+      width: 20%;
+
+      @include phone {
+        display: none;
+      }
+    }
+
+    th.c-th-amount {
+      width: 35%;
+
+      @include tablet {
+        width: 24%;
+      }
+    }
+  }
+
+  &:not(.c-is-todo) {
+    th.c-th-who {
+      width: 70%;
+
+      @include tablet {
+        width: 30%;
+      }
+    }
+
+    th.c-th-amount {
+      width: 20%;
     }
 
-    &:last-child {
-      width: 5%;
+    th.c-th-method {
+      display: none;
 
       @include tablet {
-        width: 20%;
+        display: table-cell;
+        width: 24%;
+      }
+
+      @include desktop {
+        width: 19%;
+      }
+    }
+
+    th.c-th-date {
+      width: 18%;
+      padding-right: 0.5rem;
+    }
+
+    th.c-th-relative-to {
+      display: none;
+      padding-right: 0;
+
+      @include desktop {
+        display: table-cell;
       }
     }
   }
 }
 
-.c-th-amount {
-  @include phone {
+.c-action-arrow {
+  display: inline-block;
+  margin-left: 0.5rem;
+  transform: translateY(-2px);
+}
+
+@include phone {
```

Comment:
Is it possible to add the phone special setting in the class instead of regrouping th classes in the media queries? It's easier to read later